wpdash.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: ODM Dashboards
  * Plugin URI: http://github.com/OpenDevelopmentMekong/wpdash
  * Description: Internal wordpress plugin for exposing widgets with data visualizations.
- * Version: 2.1.2
+ * Version: 2.1.3
```

Comment:
@DragonVirus FYI: please note that all PRs should include an increase on the version number.